ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -0,0 +1,112 @@
+
+
+@nameable_op
+def deformable_convolution(
+        data: NodeInput,
+        offsets: NodeInput,
+        filters: NodeInput,
+        strides: List[int],
+        pads_begin: List[int],
+        pads_end: List[int],
+        dilations: List[int],
+        modulation_scalars: Optional[NodeInput] = None,
+        auto_pad: str = "EXPLICIT",
+        group: int = 1,
+        deformable_group: int = 1,
+        bilinear_interpolation_pad: bool = False,
+        name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs deformable convolution operation.
+
+    @param data: The node providing data batch tensor.
+    @param offsets: The node providing offset tensor.
+    @param filters: The node providing filters tensor.
+    @param strides: The distance (in pixels) to slide the filter on the feature map over the axes.
+    @param pads_begin: The number of pixels to add to the beginning along each axis.
+    @param pads_end: The number of pixels to add to the end along each axis.
+    @param dilations: The distance in width and height between elements (weights) in the filter.
+    @param modulation_scalars: The node providing modulation scalar (mask) tensor.
+    @param auto_pad: The type of padding. Range of values: explicit, same_upper, same_lower, valid.
+    @param group: The number of groups which both output and input should be split into.
+    @param deformable_group: The number of groups which deformable values and output should be split
+                             into along the channel axis.
+    @param bilinear_interpolation_pad: The flag that determines the mode of bilinear interpolation
+                                               execution.
+    @param name: The optional new name for output node.
+    @return New node performing deformable convolution operation.
+    """
+    if modulation_scalars is None:
+        return _get_node_factory_opset8().create(
+            "DeformableConvolution",
+            as_nodes(data, offsets, filters),
+            {
+                "strides": strides,
+                "pads_begin": pads_begin,
+                "pads_end": pads_end,
+                "dilations": dilations,
+                "auto_pad": auto_pad,
+                "group": group,
+                "deformable_group": deformable_group,
+                "bilinear_interpolation_pad": bilinear_interpolation_pad
+            },
+        )
+    else:
+        return _get_node_factory_opset8().create(
+            "DeformableConvolution",
+            as_nodes(data, offsets, filters, modulation_scalars),
+            {
+                "strides": strides,
+                "pads_begin": pads_begin,
+                "pads_end": pads_end,
+                "dilations": dilations,
+                "auto_pad": auto_pad,
+                "group": group,
+                "deformable_group": deformable_group,
+                "bilinear_interpolation_pad": bilinear_interpolation_pad
+            },
+        )
```

Comment:
This is correct, however I would like to see solution similar to (just for example) `non_max_suppression` or `dft` ops. Always return Node but change inputs and attributes accordingly to None values.

Reply:
I'm not sure it's a good idea, we have 2 separate constructors in ngraph (with/without mask input), so we do not provide any values in mask unless it was provided by user itself. It helps us to avoid unnecessary multiplication.

Reply:
Discussed offline

Reply:
done